sale_order_type/views/sale_order_type_view.xml

```diff
@@ -0,0 +1,119 @@
+                <search string="Type">
+                    <field name="name"/>
+                    <field name="warehouse_id"/>
+                    <field name="sequence_id"/>
+                    <field name="journal_id"/>
+                    <field name="refund_journal_id"/>
+                    <field name="picking_policy" />
+                    <field name="order_policy" />
+                    <field name="invoice_state" />
+                    <group expand="0" string="Group By">
+                        <filter string="Warehouse" domain="[]"
+                                context="{'group_by':'warehouse_id'}" />
+                        <filter string="Entry Sequence" domain="[]"
+                                context="{'group_by':'sequence_id'}" />
+                        <filter string="Billing Journal" domain="[]"
+                                context="{'group_by':'journal_id'}" />
+                        <filter string="Refund Billing Journal" domain="[]"
+                                context="{'group_by':'refund_journal_id'}" />
+                        <filter string="Shipping Policy" domain="[]"
+                                context="{'group_by':'picking_policy'}" />
+                        <filter string="Create Invoice" domain="[]"
+                                context="{'group_by':'order_policy'}" />
+                        <filter string="Invoice Control" domain="[]"
+                                context="{'group_by':'invoice_state'}" />
+                    </group>
+                </search>
+            </field>
+        </record>
+
+        <record id="sot_sale_order_type_action" model="ir.actions.act_window">
+            <field name="name">Typology</field>
+            <field name="res_model">sale.order.type</field>
+            <field name="view_mode">form,tree</field>
+            <field name="view_id" ref="sot_sale_order_type_tree_view"/>
+            <field name="help" type="html">
+                <p class="oe_view_nocontent_create">Click to set a new type for sale order.</p>
+            </field>
+        </record>
+
+        <record id="sot_sale_order_type_action_form" model="ir.actions.act_window.view">
+            <field name="sequence" eval="2"/>
+            <field name="view_mode">form</field>
+            <field name="view_id" ref="sot_sale_order_type_form_view"/>
+            <field name="act_window_id" ref="sot_sale_order_type_action"/>
+        </record>
+
+        <record id="sot_sale_order_type_action_tree" model="ir.actions.act_window.view">
+            <field name="sequence" eval="1"/>
+            <field name="view_mode">tree</field>
+            <field name="view_id" ref="sot_sale_order_type_tree_view"/>
+            <field name="act_window_id" ref="sot_sale_order_type_action"/>
+        </record>
+
+        <menuitem id="sot_sale_order_type_menu" parent="base.menu_sales"
+                  action="sot_sale_order_type_action" name="Typology" sequence="20"/>
```

Comment:
This menu should be renamed to "Sale order types" and include it inside Sales > Configuration